server/meta.jsx

```diff
@@ -37,9 +40,22 @@ function validatePaymentsSDKUrl({ pathname, query, hash }) {
             throw new TypeError(`Unexpected non-string key for sdk url: ${ key }`);
         }
 
-        if (!val.match(/^[a-zA-Z0-9_,-@.]+$/) && !val.match(/^\*$/)) {
+        if (!val.match(/^[a-zA-Z0-9+_,-@.]+$/) && !val.match(/^\*$/)) {
             throw new Error(`Unexpected characters in query key for sdk url: ${ key }=${ val }`);
         }
+
+        if (key === SDK_QUERY_KEYS.MERCHANT_ID) {
+            const merchantValues = val.split(",");
+            Array.from(merchantValues).forEach(aMerchantValue => {
+                if (!emailMaxLengthRegex.test(aMerchantValue)) {
+                    throw new Error(`Email is too long: ${aMerchantValue}`)
+                }
+                if (aMerchantValue.includes("@") && !emailRegex.test(aMerchantValue)) {
```

Comment:
if it gets here wouldnt it have already passed the regex which requires `@`?

Reply:
Good point @westeezy. We don't need to check for `@` twice. 
<img width="305" alt="Screen Shot 2021-11-29 at 1 36 58 PM" src="https://user-images.githubusercontent.com/534034/143931358-a00e21f5-498b-4e64-ab4b-d0245cf5eae4.png">
e.

@asquiroga can you update it so it only has one check? My initial thought is to get rid of `aMerchantValue.includes("@")`.